README.md

```diff
@@ -1 +1,5 @@
-# security-audits
+# Security Audits
+
+| Date           | Auditor           | Title                                                                                                |
```

Comment:
Let's add a `crates` column that specifies which crates where audited as part of that audit. We are not monorepo, so this makes sense

Reply:
On it.